packages/astro/src/@types/astro.ts

```diff
@@ -918,7 +932,27 @@ export interface AstroUserConfig {
 	 * Astro offers experimental flags to give users early access to new features.
 	 * These flags are not guaranteed to be stable.
 	 */
-	experimental?: object;
+	experimental?: {
+		/**
+		 * @docs
+		 * @name experimental.assets
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 2.1.0
+		 * @description
+		 * Enable experimental support for enhanced images support and ability to optimize and resize images directly in Astro. With this enabled, a new `astro:assets` module will be exposed
```

Comment:
Does this read OK? I’m trying to avoid the double “support”.

```suggestion
		 * Enable experimental image processing support, including the ability to optimize and resize images directly in Astro. With this enabled, a new `astro:assets` module will be exposed.
```

Reply:
It’s also not clear here what you enable _apart_ from optimisation/resizing, so we could also just compress it down to that:

```suggestion
		 * Enable experimental support for optimizing and resizing images. With this enabled, a new `astro:assets` module will be exposed.
```